app/res/layout-land/activity_crash_warning.xml

```diff
@@ -93,12 +93,11 @@
         </LinearLayout>
     </ScrollView>
 
-    <Button
+    <com.google.android.material.button.MaterialButton
+        style="@style/CustomButtonStyle"
```

Comment:
It's better to apply the button style globally in the app's theme and only set it to specific buttons when a different style is needed